test-defaults.ini

```diff
@@ -63,6 +63,7 @@ arcade = "Arcade"
 console = "Console"
 arcade = "Arcade"
 con_ops = "Fest Ops"
+food_prep = "Staff Suite"
```

Comment:
For testing purposes, I added the staff suite job location as it appears in our production config.